pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// MkdirAll implementation that is volume path aware for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
+		return nil
+	}
+
+	// The rest of this method is copied from os.MkdirAll and should be kept
+	// as-is to ensure compatibility.
```

Comment:
I might be missing something here, but even after updating to 1.4.2, these implementations are the same for me.  The spacing on PathError is different (I put each argument on it's own line) and since I'm not in the os package I need to explicitly add "os." in front of the methods that come out of there.  Other than that, it is identical, as you can see from comparing it to the link you posted.